NOMIS/nomis.go

```diff
@@ -109,13 +116,32 @@ func main() {
 		return
 	}
 
-	for index0, _ := range res.Structure.Keyfamilies.Keyfamily {
+	for index0 := range res.Structure.Keyfamilies.Keyfamily {
+		var annotations = res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation
 		censusEditionData := models.EditionUpdate{}
+		censusInstances := models.Version{}
 		mapData := models.Dataset{}
-		cenId := res.Structure.Keyfamilies.Keyfamily[index0].ID
-		mapData.Title = res.Structure.Keyfamilies.Keyfamily[index0].Name.Value
-		mapData.ID = cenId
+		var cenId string
+		for censusId := range annotations {
+			annoIndex := res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[censusId]
+			switch annoIndex.Title {
+			case "Mnemonic":
+				ref := annoIndex.Text.(string)
+				cenParam := strings.Split(ref, "c2011")
+				if len(cenParam) < 2 {
+					log.Event(ctx, "error Mnemonic length invalid", log.ERROR)
```

Comment:
Errors in Go should contain punctuation or capital letters (M should be lowercase here)